test/helpers/constants.ts

```diff
@@ -1,7 +1,22 @@
+// constants.ts -  Any common values here should be moved to moonwall if suitable
+
+import {
+  ALITH_GENESIS_FREE_BALANCE,
+  ALITH_GENESIS_LOCK_BALANCE,
+  ALITH_GENESIS_RESERVE_BALANCE,
+} from "@moonwall/util";
+
 // Crowdloan Constants
 
 export const VESTING_PERIOD = 201600n;
 export const RELAYCHAIN_ARBITRARY_ADDRESS_1: string =
   "0x1111111111111111111111111111111111111111111111111111111111111111";
 export const RELAYCHAIN_ARBITRARY_ADDRESS_2: string =
   "0x2222222222222222222222222222222222222222222222222222222222222222";
+
+export const ALITH_GENESIS_TRANSFERABLE_COUNT =
+  ALITH_GENESIS_FREE_BALANCE + ALITH_GENESIS_RESERVE_BALANCE - ALITH_GENESIS_LOCK_BALANCE;
+export const ALITH_GENESIS_TRANSFERABLE_BALANCE =
+  ALITH_GENESIS_FREE_BALANCE > ALITH_GENESIS_TRANSFERABLE_COUNT
+    ? ALITH_GENESIS_TRANSFERABLE_COUNT
+    : ALITH_GENESIS_FREE_BALANCE;
```

Comment:
Why in `helpers/constants.ts`? Seems inconsistent and perhaps a bit noisy in the tests. Not really a big deal though.

Reply:
This is the preferred place to put project specific constants, which is relevant here given the change of lock behaviour now means the default alith balance has changed.

Reply:
Sounds good to me, does this deserve a refactor PR to make things more consistent?